spec/bolt/outputter/human_spec.rb

```diff
@@ -255,6 +255,18 @@
     expect(output.string).to match(/Finished on #{target}.*value/m)
   end
 
+  it "doesn't stacktrace when merged_output is nil" do
+    value = {
+      'stdout'        => 'stdout',
+      'stderr'        => 'stderr',
+      'merged_output' => nil,
+      'exit_code'     => 2
+    }
+    outputter.print_result(Bolt::Result.for_command(target, value, 'command', "executed", []))
+    expect { output.string }.not_to raise_error
```

Comment:
We want to wrap the `expect` block around the part of the code that we want to ensure does not error, which is the line above (`outputter.print_result`).

Reply:
@artlawson I think this got missed.